enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -2865,6 +2916,10 @@ func (e *partitionEvictor) deleteFile(key filestore.PebbleKey, version filestore
 		}
 	case storageMetadata.GetInlineMetadata() != nil:
 		break
+	case storageMetadata.GetGcsMetadata() != nil:
```

Comment:
does the logic around `storedSizeBytes` below do what you want it to in this case?  seems like the existing logic is all for local files, whereas maybe we don't want this to count that way?